### ISSUE TITLE
End-points Fixed

### ISSUE THREAD

**Issue by sperezconesa** [delemottelab/string-method-gmxapi#13]
Hello Oliver,

If the endpoints are fixed, does this mean that no restrained simulations are run from those points? How do you sample the region around the endpoints just by swarms that fall into them. Would it be possible to online have the restrained simulations at the endpoints or is this not worth it? 

Thanks!
Sergio

**Comment by oliverfleetwood** [comment]
Yes, if the endpoints fixed you won't have restrained simulations. 
You can sample the area around them by either releasing the endpoints or having enough points on the string for the adjacent points to almost overlap. 

Note that in practice you don't know exactly where the basins are, and the endpoints may be in high free energy regions. In such cases you probably don't need to sample them anyway.

**Comment by oliverfleetwood** [comment]
I think it's not worth to have restrained simus at the endpoints but that can be discussed of course.
